### DISCUSSION TITLE
tutorials/intro-strawberry/01-overview-setup

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#134] [course feedback]
# tutorials/intro-strawberry/01-overview-setup

Learn what you're building in this Introduction to GraphQL and set up your project.

https://www.apollographql.com/tutorials/intro-strawberry/01-overview-setup

**Comment by R4ph-t** [comment]
Woohoo a new language supported on Odyssey!

**Comment by bparanj** [comment]
It would be nice to mention using insomnia for the front end instead of writing code for client side functionality. The reason is that the core essence of this should be building the schema and creating a quick prototype for iterating fast.

**Reply by mabuyo** [reply]
Hi there! To clarify, we won't be implementing anything on the frontend side, but we'll use mockup designs to guide our schema creation.
